pglet/grid.py

```diff
@@ -128,8 +134,8 @@ def max_width(self):
         return self._get_attr("maxWidth")
 
     @max_width.setter
-    def max_width(self, value):
-        assert value == None or isinstance(value, int), "maxWidth must be an int"
+    @beartype
+    def max_width(self, value: Optional[int]):  # could these not be floats? Union[None, int, float]
```

Comment:
So, in Fluent's UI `IColumn` definition it's TypeScript's [`number`](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DetailsList/DetailsList.types.ts#L396-L397) which, well, could be int or float. I've never tested it with something other than int though.

Reply:
Without any facts I have a strong feeling FluentUI would accept all dimension and coordinate values as floats, given the javascript/HTML DOM foundation. Thus it would likely make sense at least not fail loudly on getting floats.